src/core/state.go

```diff
@@ -874,6 +874,19 @@ func (state *BuildState) queueTarget(label, dependent BuildLabel, rescan, forceB
 	return nil
 }
 
+func (state *BuildState) QueueTestTarget(target *BuildTarget) {
+	go state.queueTestTarget(target)
```

Comment:
not sure if you need to do something about the count of pending actions or not here? maybe it has already been taken care of?